.circleci/deploy-python-2.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/sh
 
 set -e
-if ! git rev-parse `python setup.py --version` 2>/dev/null ; then
+
+if ./detect-functional-changes.sh
+then
     git tag `python setup.py --version`
     git push --tags  # update the repository version
```

Comment:
I don't see the difference between Python2 and Python3 deployment beyond this initial tagging.
If indeed there is none, then I think the tagging should be done in CI (or in a script), and that the deployment should be a single script as it was. Right now, there is an apparent dependency of deployment of Python3 to deployment of Python2, which is both absurd and not guaranteed by CI; there is also unnecessary code duplication in the deployment scripts.

Reply:
Done